bin/zeppelin-daemon.sh

```diff
@@ -67,12 +67,17 @@ if [[ -d "${ZEPPELIN_HOME}/zeppelin-server/target/classes" ]]; then
   ZEPPELIN_CLASSPATH+=":${ZEPPELIN_HOME}/zeppelin-server/target/classes"
 fi
 
-addJarInDir "${ZEPPELIN_HOME}"
-addJarInDir "${ZEPPELIN_HOME}/lib"
-addJarInDir "${ZEPPELIN_HOME}/zeppelin-interpreter/target/lib"
-addJarInDir "${ZEPPELIN_HOME}/zeppelin-zengine/target/lib"
-addJarInDir "${ZEPPELIN_HOME}/zeppelin-server/target/lib"
-addJarInDir "${ZEPPELIN_HOME}/zeppelin-web/target/lib"
+addEachJarInDir "${ZEPPELIN_HOME}"
+addEachJarInDir "${ZEPPELIN_HOME}/lib"
+addEachJarInDir "${ZEPPELIN_HOME}/zeppelin-interpreter/target/lib"
+addEachJarInDir "${ZEPPELIN_HOME}/zeppelin-zengine/target/lib"
+addEachJarInDir "${ZEPPELIN_HOME}/zeppelin-server/target/lib"
+addEachJarInDir "${ZEPPELIN_HOME}/zeppelin-web/target/lib"
```

Comment:
@gliptak Appreciate for the contribution!

About generating classpath, because of the problem such as [ZEPPELIN-68](https://issues.apache.org/jira/browse/ZEPPELIN-68), addJarInDir() is introduced and used intentionally by this [commit](https://github.com/apache/incubator-zeppelin/commit/dcb03a9fe66d753348154c41f5b40927125723aa).

So, will there be any way to detect Zeppelin is not been built without change classpath generation?